assets/styles/style.css

```diff
@@ -0,0 +1,188 @@
+:root {
+  /* color */
+  --clr-primary: hsl(222, 11%, 17%);
+  --clr-secondary: hsl(7, 84%, 58%);
+  --clr-primary-light: hsl(0, 0%, 27%);
+  --clr-neutral: hsl(0, 0%, 83%);
+  --clr-neutral-light: hsl(0, 0%, 95%);
+  --clr-white: hsl(0, 0%, 100%);
```

Comment:
- [ ] Please 🙏🏽 use the HEX format in the colors and the corresponding colors from the original design:
- \#272A31
- \#EC5242
- \#D3D3D3

Reply:
thanks @franclobo   These are the same as the HEX format, should i change them even they are the same?

Reply:
Yes please it is easier to read that transform to HSL or use inline comments to describe the equivalent in HEX.